js/data/bucket/symbol_bucket.js

```diff
@@ -61,11 +61,11 @@ var layoutVertexArrayType = new Bucket.VertexArrayType([{
     type: 'Int16'
 }, {
     name: 'a_data1',
```

Comment:
What do you think about renaming this to something more self-documenting like `a_texture_pos`?